pages/Documents/RichMessaging/AppleMessagesForBusinessTemplates/form-message-template.md

```diff
@@ -0,0 +1,583 @@
+---
+pagename: Form Message Template
+sitesection: Documents
+categoryname: "Rich Messaging"
+documentname: Apple Messages for Business Templates
+permalink: apple-messages-for-business-templates-form-message-template.html
+indicator: messaging
+---
+
+### Overview
+
+[Messages for Business Forms](https://register.apple.com/resources/messages/msp-rest-api/type-interactive#form-message) allow you to create rich, multi-page interactive flows for users on iOS and iPadOS devices using a single JSON payload.
+This allows brands to collect large amounts of information from customers using a seamless interface that is contained within the Apple Messaging Interface. This can allow for a wide range of interactivity without requiring the consumer to leave the conversation.
+
+Sending the structured content templates (body and metadata) explained in this document will allow you to send Business Forms to the consumer device. 
+
+For instructions on how to send structured content with metadata in Conversational Builder, please follow the steps [here.](https://developers.liveperson.com/conversation-builder-interactions-code.html#updating-conversation-metadata)
```

Comment:
Please rephrase (not to use a “here” link), as with “follow the steps for…” or ”…to…”.

Reply:
Implemented requested change